airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/NotificationsForm.tsx

```diff
@@ -59,6 +60,7 @@ const NotificationsForm: React.FC<NotificationsFormProps> = ({
           name="newsletter"
```

Comment:
Is `LabeledToggle` a wrapper on `Switch`?  Should it be `LabeledSwitch`?

Reply:
Oh yes! I'll change